### DISCUSSION TITLE
인기글 기준 관련

### DISCUSSION THREAD

**Discussion by cho-ja-young** [MajorIn-ABO/MajorIn_FE#8] [General]
인기글 기준을 조회수로 하기로 했는데, 생각해보니까
좋아요 개수나 북마크를 기준에 넣는 것도 생각해봐야 할 것 같아 작성했음!

같은 조회수인데 좋아요나 북마크가 많으면 더 앞쪽으로 정렬시키는 우리만의 알고리즘이 필요할 것 같음!
일단은 조회수로만 하고 (복잡하니께) 이거 관련한 인기글 알고리즘을 우리가 짜서 반영시키는거 어때?

**Comment by JangGusWjd** [comment]
오 아니면 일단 커뮤니티나 스터디는 글이 올라온 순서가 중요하니까 일단 디폴트가 최신순으로 하고 그 옵션 메뉴바?그걸로 인기순, 좋아요 순 정렬 메뉴 만들어서 인기순은 조회수 높은 순, 따로 좋아요 많은 순? 이렇게 하는 것도 괜찮을 거 같기도! 그 메인에 있는 실시간 인기글을 우리만의 알고리즘 적용하는 것도 좋을거같아!

**Comment by wlsl10** [comment]
데이터 양이 너무 많으면 또 문제가 되니까 일단 조회수로 정렬하고, 조회수가 같은 경우 좋아요가 높은 순으로 하면 좋을 것 같아서, 좋아요 수까지만 데이터베이스에 저장해두면 좋을 것 같아! 그리고 현정이 말처럼 스터디는 디폴트가 최신순이고, 좋아요 수 혹은 조회수로 높은 순으로 알고리즘 적용하는 게 좋을 것 같아! 너무 복잡하면 그냥 스터디도 기존 방식과 동일하게 해도 될 것 같아!
